packages/devtools_app/lib/src/framework/status_line.dart

```diff
@@ -159,7 +159,7 @@ class StatusLine extends StatelessWidget {
               DevToolsTooltip(
                 message: 'Device Info',
                 child: InkWell(
-                  onTap: () async {
+                  onTap: () {
                     unawaited(
```

Comment:
if the callback is no longer async, do we still need to mark this unawaited?

Reply:
yes, otherwise lint is complaining that we drop the future